### ISSUE TITLE
Load tests CLI

### ISSUE THREAD

**Issue by kevsul** [omisego/elixir-omg#1444]
It would be nice to be able to run the load tests via CLI, passing params as command line args

**Comment by boolafish** [comment]
we have quite a lot a params, I think from a file might be better.

**Comment by kevsul** [comment]
> we have quite a lot a params, I think from a file might be better.

Yeah, I'm thinking a config file that holds the majority of vars such as urls, faucet key, etc. but that can be overridden with command line args such as number of txs, users, delays...
